go/cmd/vtbackup/vtbackup.go

```diff
@@ -120,6 +121,11 @@ var (
 	detachedMode     bool
 	keepAliveTimeout = 0 * time.Second
 	disableRedoLog   = false
+	durationByPhase  = stats.NewGaugesWithSingleLabel(
+		"duration_seconds",
+		"How long it took vtbackup to perform a each phase of operation (in seconds).",
```

Comment:
Because vtbackup also picks up stats from `mysqlctl`, there will be some overlap between this metric and those metrics. I think that's OK, personally, but if we want I can take more care to avoid any overlap.